README.md

```diff
@@ -14,65 +14,65 @@ the library in ad-hoc or CI shell scripting outside of Go.
-   - **Cons:**
-     - Installation may consume some bandwidth, disk space and a little time
-     - Potentially less stable builds (see `checkpoint` below)
- - `checkpoint.LatestVersion` - Downloads, verifies & installs any known product available in HashiCorp Checkpoint
-   - **Pros:**
-     - Checkpoint typically contains only product versions considered stable
-   - **Cons:**
-     - Installation may consume some bandwidth, disk space and a little time
-     - Currently doesn't allow installation of old versions or enterprise versions (see `releases` above)
- - `build.GitRevision` - Clones raw source code and builds the product from it
-   - **Pros:**
-     - Useful for catching bugs and incompatibilities as early as possible (prior to product release).
-   - **Cons:**
-     - Building from scratch can consume significant amount of time & resources (CPU, memory, bandwith, disk space)
-     - There are no guarantees that build instructions will always be up-to-date
-     - There's increased likelihood of build containing bugs prior to release
-     - Any CI builds relying on this are likely to be fragile
+  - **Cons:**
+    - Only relies on a single version, expects _you_ to manage the installation
+    - _Not recommended_ for any environment where product installation is not controlled or managed by you (e.g. default GitHub Actions image managed by GitHub)
+- `releases.{LatestVersion,ExactVersion}` - Downloads, verifies & installs any known product from `releases.hashicorp.com`
+  - **Pros:**
+    - Fast and reliable way of obtaining any pre-built version of any product
+    - Allows installation of enterprise versions
+  - **Cons:**
+    - Installation may consume some bandwidth, disk space and a little time
+    - Potentially less stable builds (see `checkpoint` below)
+- `checkpoint.LatestVersion` - Downloads, verifies & installs any known product available in HashiCorp Checkpoint
+  - **Pros:**
+    - Checkpoint typically contains only product versions considered stable
+  - **Cons:**
+    - Installation may consume some bandwidth, disk space and a little time
+    - Currently doesn't allow installation of old versions or enterprise versions (see `releases` above)
+- `build.GitRevision` - Clones raw source code and builds the product from it
+  - **Pros:**
+    - Useful for catching bugs and incompatibilities as early as possible (prior to product release).
+  - **Cons:**
+    - Building from scratch can consume significant amount of time & resources (CPU, memory, bandwith, disk space)
+    - There are no guarantees that build instructions will always be up-to-date
+    - There's increased likelihood of build containing bugs prior to release
+    - Any CI builds relying on this are likely to be fragile
 
 ## Example Usage
 
-See examples at https://pkg.go.dev/github.com/hashicorp/hc-install#example-Installer.
+See examples at <https://pkg.go.dev/github.com/hashicorp/hc-install#example-Installer>.
```

Comment:
I struggle to find the meaning of this `<URL>` syntax - do you mind sharing link to any relevant docs to help me understand the reason for the change?

Reply:
I use [markdownlint](https://github.com/DavidAnson/markdownlint/tree/main) to format Markdown
- The rule for the "no bare URL's" is MD034 - https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md034.md